app/controllers/WhatIsNextController.scala

```diff
@@ -60,19 +63,31 @@ class WhatIsNextController @Inject()(
   def onSubmit(): Action[AnyContent] = actions.authenticatedForUtr.async {
     implicit request =>
 
+      def clearTransforms(value: WhatIsNext): Future[Any] = {
```

Comment:
Is there a way to have a unified result instead of Future[Any]. At the very least, the Future of Unit worries me. :)